<launch>
  <node pkg="robot_bringup" name="robot_bringup_node" type="robot_bringup_node" output="screen">
    <param name="uart_id" type="string" value="/dev/Tof-485"/>
    <param name="vel_topic" type="string" value="/cmd_vel"/>
    <param name="wheelbase" type="double" value= "0.52"/>
  </node>

  <node pkg="robot_bringup" name="pose2odom" type="pose2odom" output="screen">
  </node>

  <include file="$(find lpms_ig1)/launch/lpmsig1.launch" />
  
  <!--- rslidar2velodyne转换节点 -->
  <!--include file="$(find robot_bringup)/launch/include/rs_to_vlp.launch" /-->

  <!--- rslidar sdk -->
  <!--include file="$(find robot_bringup)/launch/include/rslidar.launch" /-->

  <!--- IMU node -->
  <!--include file="$(find robot_bringup)/launch/include/lpms_imu.launch" /-->
  <!--include file="$(find lpms_ig1)/launch/lpmsig1.launch" /-->
</launch>
